src/Compression/CompressionInfo.h

```diff
@@ -46,7 +46,6 @@ enum class CompressionMethodByte : uint8_t
     AES_128_GCM_SIV = 0x96,
     AES_256_GCM_SIV = 0x97,
     FPC             = 0x98,
-    DeflateQpl      = 0x99,
     GCD             = 0x9a,
     ZSTD_QPL        = 0x9b,
```

Comment:
We will accept adding it back, but you also have to fix these architectural mistakes.

1. While ZSTD_QPL has the same format as ZSTD, it uses a different byte identifier for no reason, and files compressed with this codec cannot be decompressed on the systems without QPL.

2. It is named QPL, while the library is named QAT.

3. Too many individual boolean flags in function signatures.

Reply:
> While ZSTD_QPL has the same format as ZSTD, it uses a different byte identifier for no reason, and files compressed with this codec cannot be decompressed on the systems without QAT.

The different byte identifier is because standard ZSTD supports compression levels 1-22 but ZSTD-QAT supports only levels 1-12 (hardware limitations ...). But yes, I think we should instead have _one_ ZSTD byte identifier and make both as interoperable as possible (there could be some best-effort decompression).